lib/manager-tasks/getGroupsTask.ts

```diff
@@ -16,10 +17,26 @@ task(MANAGER_GET_GROUPS, MANAGER_GET_GROUPS_TASK_DESCRIPTION)
       taskLogger.info("Starting stakedCelo:manager:getGroups task...");
       await setLocalNodeDeploymentPath(hre);
 
-      const managerContract = await hre.ethers.getContract("Manager");
-      const deprecatedGroups = await managerContract.getGroups();
-      taskLogger.log("Groups:", deprecatedGroups);
+      const defaultStrategyContract = await hre.ethers.getContract("DefaultStrategy");
+      const groups = await getDefaultGroupsSafe(defaultStrategyContract)
+      taskLogger.log("Groups:", groups);
     } catch (error) {
       taskLogger.error("Error getting groups:", error);
     }
   });
+
+  export async function getDefaultGroupsSafe(
```

Comment:
There main reason is that once we allow more than 10 validator for our Account contract, technically we might have all validator groups activated. In such case `activeGroups.getElements()` might run out of gas and we would have no way how to get all active groups. For that reason we would still need to have function with index based approach. Originally I wanted to keep both function but then I ran into issue with smart contract size. And since public/external functions take rather lots of byte space I decided to remove this function. Also it felt right to keep smart contract without any redundancies.